addon/templates/components/frost-modal-dialog.hbs

```diff
@@ -82,6 +83,7 @@
       priority=button.priority
       size='medium'
       onClick=button.onClick
+      tabIndex=(ifUndefined button.tabIndex 1 button.tabIndex)
```

Comment:
it seems this would give all custom buttons a tabIndex of 1 if that prop is undefined?